app/src/main/java/com/globant/practice/presentation/view/activity/HomeActivity.java

```diff
@@ -47,4 +68,60 @@ protected void onPause() {
         super.onPause();
         presenter.detachView();
     }
+
+    /**
+     * Renders the view items depending of the view state
+     *
+     * @param homeViewState State of the view
+     */
+    @Override
+    public void render(HomeViewState homeViewState) {
+        if (homeViewState.isLoading()) {
+            fetchUserIndicator.show();
+        } else if (homeViewState.getUsers() != null) {
+            listHomeRecyclerView.setHasFixedSize(true);
+            listHomeAdapter.setAdapterData(homeViewState.getUsers());
+            listHomeRecyclerView.setAdapter(listHomeAdapter);
```

Comment:
setAdapter, setDecorator, LayoutManager, etc. can be done right after creating the adapter above, there is no need in re-setting every time render method is invoked